packages/plugin-full-text-search/src/server/searchField.ts

```
@@ -0,0 +1,114 @@
+  }
+
+  const newCollection = db.getCollection(foreignField.target);
+  const newField = parts.join('.'); // 剩余部分
+  return getCollectionField(newCollection, newField, db);
+}
+
+export function handleField(
+  handler: FieldBase,
+  func: Function,
+  field: string,
+  fields: Map<string, any>,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)